tests/playwright/pages/editor-page.ts

```diff
@@ -790,7 +790,7 @@ export default class EditorPage extends BasePage {
 		await this.page.locator( EditorSelectors.panels.siteSettings.wrapper ).waitFor();
 
 		if ( innerPanel ) {
-			await this.page.locator( `.elementor-panel-menu-item-settings-${ innerPanel }` ).click();
```

Comment:
We did the same with Pro tests. It helps open not only the "setting" panels in Site Settings, but all the panels including global colors and global fonts.

Simple usage: `editor.openSiteSettings( innerPanel );`

We don't need to reinvent the logic in other places, simply use this primitive method.